app/models/concerns/user_role.rb

```diff
@@ -42,6 +43,10 @@ def service_user?
     [CASEWORKER, SUPERVISOR].include?(role) && !can_manage_others?
   end
 
+  def reporting_user?
+    [SUPERVISOR, DATA_ANALYST].include?(role) && !can_manage_others?
```

Comment:
I'm pretty sure a supervisor should also be able to access the service. Just the data analyst that can't - needs double checking